### PR TITLE
[RFC] Update version.c to latest Vim

### PR DESCRIPTION
**Pull request by Shougo**
I have updated version.c to latest Vim(7.4.922).
And I have added following patches as NA.

Please review it.

NA patches list:

```
652: xxd changes  xxd is not included in neovim
744: if_ruby and if_perl legacy tests
750: if_perl build problem
751: compile options
756: if_perl support
757: term.c changes
759: if_lua support
761: term.c changes
762: term.c changes
763: if_lua support
766: term.c changes
770: term.c changes
772: if_mzsch support
802: duplicated test
809: revert of 802
829: GUI
837: Windows with if_sniff
840: GUI
846: CONTRIBUTING.md
850: CONTRIBUTING.md
851: Win32 console
852: Win32 console
855: GTK GUI
861: GTK GUI
862: GTK GUI
867: Windows Fix
869: Windows GUI
872: CI service
874: Windows GUI
875: README.md
876: Windows GUI
880: CI service
884: CI service
886: Windows GUI
890: if_python3
894: vimrun.exe
899: README.md
900: README.md
902: Win32 console
906: Windows problem
907: Dynamic loading
908: Windows build error
910: if_python
911: terminal
913: hangul input GUI only
918: if_python
919: if_lua, if_python, if_python3
921: Windows
```

Edit:  Removed 916 from NA

### CODE REVIEW

**Comment by ZyX-I** [comment]
About 916: https://github.com/neovim/neovim/pull/3630#discussion_r44524810.

**Comment by Shougo** [comment]
@ZyX-I Thanks!  I will update it.

**Comment by Shougo** [comment]
Rebased and updated.

**Comment by Pyrohh** [comment]
@Shougo could you please rebase?

**Comment by Shougo** [comment]
Thanks.  I will do it.

**Comment by Shougo** [comment]
Merged and rebased to latest neovim.

File: src/nvim/version.c [hunk 1]
```diff
@@ -74,86 +74,92 @@ static char *features[] = {
...
+  // 886 NA
   // 885,
-  // 884,
+  // 884 NA
   // 883,
   // 882,
   // 881,
   // 880,
-  // 879,
+  // 879 NA
```
Comment (Pyrohh) [inline src/nvim/version.c]: Why was this marked NA? It isn't specified in the commit message, and, looking at Nvim's commit log, it doesn't seem to be included.

Oh, and when you repush please don't amend the commit; I'll do that before merging.

Comment (Pyrohh) [inline src/nvim/version.c]: Oh, I think I get it. Did you mean to mark `880` as NA instead?

**Comment by Pyrohh** [comment]
Thanks @Shougo, commited in 1fbb56795d16783f9a97e25e3b71ab3ac2a644dc.

Here's the diff between your commit and the modified one:
```diff
--- old-version.patch	Fri Nov 20 13:45:48 2015
+++ new-version.patch	Fri Nov 20 13:45:34 2015
@@ -1,4 +1,4 @@
-From e9ab4a333f2796d54a4e5d5da4850277ad4b01bc Mon Sep 17 00:00:00 2001
+From 1fbb56795d16783f9a97e25e3b71ab3ac2a644dc Mon Sep 17 00:00:00 2001
 From: Shougo Matsushita <Shougo.Matsu@gmail.com>
 Date: Wed, 11 Nov 2015 20:40:40 +0900
 Subject: [PATCH] Update version.c to latest Vim
@@ -50,15 +50,19 @@
 910: if_python
 911: terminal
 913: hangul input GUI only
+917: hangulin.c changes
 918: if_python
 919: if_lua, if_python, if_python3
+920: ruby stuff
 921: Windows
+
+Reviewed-by: Michael Reed <Pyrohh@users.noreply.github.com>
 ---
  src/nvim/version.c | 98 +++++++++++++++++++++++++++++-------------------------
  1 file changed, 52 insertions(+), 46 deletions(-)
 
 diff --git a/src/nvim/version.c b/src/nvim/version.c
-index f5e4387..0a335c2 100644
+index f5e4387..d5bbd73 100644
 --- a/src/nvim/version.c
 +++ b/src/nvim/version.c
 @@ -74,86 +74,92 @@ static char *features[] = {
@@ -70,7 +74,7 @@
 +  // 920 NA
 +  // 919 NA
 +  // 918 NA
-+  // 917,
++  // 917 NA
    916,
    // 915,
    // 914,
@@ -120,9 +124,9 @@
    // 883,
    // 882,
    // 881,
-   // 880,
--  // 879,
-+  // 879 NA
+-  // 880,
++  // 880 NA
+   // 879,
    // 878,
    // 877,
 -  // 876,
@@ -267,3 +271,6 @@
    651,
    // 650 NA
    // 649,
+-- 
+2.6.3
+

```

**Comment by Shougo** [comment]
Thank you for the fix!

> Oh, I think I get it. Did you mean to mark 880 as NA instead?

Yes, it is my fault.

**Comment by Pyrohh** [comment]
You're welcome :-)

Next time it might be easier to break these into small patches, as this one took me a while to review.

**Pull request closed without merging**